.github/workflows/proposal-pr-check.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+   changed_files:
+    runs-on: ubuntu-latest 
+    name: Test added-OFEP proposals for status
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get added files
+        id: added-files
+        uses: tj-actions/changed-files@v37
+        with: 
+          files: |
+            **.md
```

Comment:
```suggestion
          files: |
            **.md
          files_ignore: README.md
```

Reply:
I guess  this also might be an edge case because no-one will add a readme file to the repo.

Reply:
No, I don't think this is an edge case because someone may edit the readme in the future.

Reply:
Yep, but here I am only observing the newly added files , hence readme doesn't get detected as it can only be edited.